.github/workflows/manual-run.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         with:
           module_id: server-avaiblaility-manager
           testrail_project: Server Availability Manager
+          timeout_minutes: 20
```

Comment:
In manual run you should also add the timeout on at the job level (timeout-minutes: 30)

Reply:
Fixed, thanks